Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx

```diff
@@ -175,15 +177,26 @@ const validate = ({ content, schema }) => {
     }, {});
 };
 
+function makeLink(url: string) {
+  return ({ children }) => (
+    <Link key={url} href={url} rel="noopener noreferrer" target="_blank">
```

Comment:
Would it be better to generate a unique key here? If we have the url add to multiple links later on, we might run into issues.

Reply:
The `key` prop is only here to make formatMessage stop throwing out warnings; they only need to be unique within a single formatMessage.rich call. If we ever need to put multiple links to the same URL in the same string, we can revisit this.